output.R

```diff
@@ -60,7 +60,7 @@ choose_folder <- function(folder, title = "Please choose a folder") {
   # Detect all output folders containing fulldata.gdx
   # For coupled runs please use the outcommented text block below
 
-  dirs <- sub("/fulldata.gdx", "", sub("./output/", "", Sys.glob(file.path(folder, "*", "fulldata.gdx"))))
+  dirs <- sub("/fulldata\\.gdx$", "", sub("^\\./output/", "", Sys.glob(file.path(folder, "*", "fulldata.gdx"))))
```

Comment:
This looks correct, although I find regex in general hard to read, and regex tends to produce fragile code. What about this:
```suggestion
  dirs <- basename(dirname(Sys.glob(file.path(folder, "*", "fulldata.gdx"))))
```

Reply:
Yes, I like that much better